docs/rewards/managing-rewards.md

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 2
+---
+
+# Managing Rewards 
+
+This guide will go through how to rewards are managed on the Archway Network by explaining the important fields and operations needed to track and receive rewards. 
```

Comment:
```suggestion
This guide will go through how rewards are managed on the Archway Network by explaining the important fields and operations needed to track and receive rewards.
```

Reply:
Re: Minimum Consensus Fee - Understand the importance here, I tried to keep this content solely focused on rewards. Obviously, these are connected, do you have an idea where you would like to see it specifically added? 

Re: Top Level - When I started I was told these pages are owned by the design team so I have not edited them. The config file for adding this would require me to change the docusaurus config files included in the gitignore. Please confirm if I should be the one making these changes.